packages/jsii-pacmak/test/expected.jsii-calc-lib/java/src/main/java/software/amazon/jsii/tests/calculator/lib/StructWithOnlyOptionals.java

```diff
@@ -16,112 +18,140 @@
     java.lang.Boolean getOptional3();
     void setOptional3(final java.lang.Boolean value);
 
-    // ==================================================================
-    // Builder
-    // ==================================================================
-
+    /**
+     * @return a {@link Builder} of {@link StructWithOnlyOptionals}
+     */
     static Builder builder() {
         return new Builder();
     }
 
     /**
-     * A fluent builder class for {@link StructWithOnlyOptionals}.
+     * A builder for {@link StructWithOnlyOptionals}
      */
-    public static final class Builder {
-        private Jsii$Pojo instance = new Jsii$Pojo();
+    final class Builder {
```

Comment:
Shouldn’t it be static as well?

Reply:
It's a member of an `interface`, so it is inherently static.